Sources/SmileID/Classes/Networking/Models/SecurityInfo.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+public struct SecurityInfo: Codable {
```

Comment:
```suggestion
public struct SecurityInfoRequest: Codable {
```

Reply:
It's not really a request - it's just the representation of the json file contents. So I'd leave it as `SecurityInfo`.